inbox/src/controllers/handlers/unconfirmed-email-handlers.ts

```diff
@@ -48,9 +48,17 @@ export async function storeUnconfirmedEmailHandler(
       userName = profile.avatars[0].name
     }
 
+    const validateButtonLink = !!body.isCreditsWorkflow
```

Comment:
```suggestion
    const redirect = body.redirect ? `?redirect=${encodeURIComponent(body.redirect)}` : ''
```
can we define this above and re-use it below just to make it a bit clear to read? 🙏

Reply:
yasss